packages/main/src/themes/sap_fiori_3/parameters-bundle.css

```diff
@@ -1,5 +1,6 @@
 @import "../base/sizes-parameters.css";
 @import "../base/Avatar-parameters.css";
```

Comment:
you can remove 
@import "../base/Avatar-parameters.css";

as you import
@import "./Avatar-parameters.css";